File: url-shortening-service/src/main/java/com/project/url_shortening_service/model/UrlMapping.java
package com.project.url_shortening_service.model;

public class UrlMapping {
    private final String id;
    private final String url;
    private final String actualUrl;

    public UrlMapping(String id, String url, String actualUrl) {
        this.id = id;
        this.url = url;
        this.actualUrl = actualUrl;
    }

    public String getId() {
        return id;
    }

    public String getUrl() {
        return url;
    }

    public String getActualUrl() {
        return actualUrl;
    }
}

File: url_shortening_service_flask/model/url_mapping.py
class UrlMapping:
    def __init__(self, id, url, actual_url):
        self.id = id
        self.url = url
        self.actual_url = actual_url

    def to_dict(self):
        return {
            "id": self.id,
            "url": self.url,
            "actual_url": self.actual_url
        }
